src/index.css

```diff
@@ -0,0 +1,68 @@
+:root {
```

Comment:
Remove the index.css file. Search about how to add font-family with MUI

Reply:
done